packages/dd-trace/src/appsec/reporter.js

```diff
@@ -13,54 +14,52 @@ const {
 // default limiter, configurable with setRateLimit()
 let limiter = new Limiter(100)
 
-// TODO: use precomputed maps instead
-const REQUEST_HEADERS_PASSLIST = [
-  'accept',
-  'accept-encoding',
-  'accept-language',
+const contentHeaderList = [
   'content-encoding',
   'content-language',
   'content-length',
-  'content-type',
-  'forwarded',
-  'forwarded-for',
+  'content-type'
+]
+
+const REQUEST_HEADERS_PASSMAP = mapHeaderAndTags([
+  'accept',
+  'accept-encoding',
+  'accept-language',
   'host',
-  'true-client-ip',
   'user-agent',
+
+  'forwarded',
   'via',
-  'x-client-ip',
-  'x-cluster-client-ip',
-  'x-forwarded',
-  'x-forwarded-for',
-  'x-real-ip'
-]
 
-const RESPONSE_HEADERS_PASSLIST = [
-  'content-encoding',
-  'content-language',
-  'content-length',
-  'content-type'
-]
+  ...ipHeaderList,
+  ...contentHeaderList
+])
+
+const RESPONSE_HEADERS_PASSMAP = mapHeaderAndTags([
+  ...contentHeaderList
+], 'http.response.headers.')
+
+function mapHeaderAndTags (headerList, tagPrefix = 'http.request.headers.') {
```

Comment:
I'd rather have `http.request.headers.` be explicit in the first call, than have a default case like this. It's not like this function is going to have a lot of different use cases.